src/stores/activityStore.ts

```diff
@@ -15,19 +12,11 @@ export class ActivityStore {
   @observable commonCards = ["", "", "", "", ""];
   @observable playersCards: string[][] = [["", ""]];
 
-  @action setCommonCards(newCards: string[]) {
-    this.commonCards = newCards;
+  @action setCommonCards(newCards: CardArrayInterface[]) {
+    this.commonCards = changeCardNaming([newCards])[0] as string[];
```

Comment:
Please create a type specific for common cards, even if it is a string array. If you'll change it later you would have all places for that.
You can create new file in `src/types` and name it something like `type CommonCards = string[];`